contracts/deploy/sign.py

```diff
@@ -59,7 +66,8 @@ def format_int(value, size):
             msg += arg
         elif isinstance(arg, str):
             if is_0x_prefixed(arg):
-                msg += decode_hex(arg)
+                arg = pad_left(remove_0x_prefix(arg), 64, '0')
+                msg += decode_hex(add_0x_prefix(arg))
```

Comment:
adding the prefix again isn't necessary, `decode_hex` handles strings with and without prefixes